ballerina/types.bal

```diff
@@ -0,0 +1,147 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+// This file is auto-generated by the Ballerina OpenAPI tool.
+
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+# Represents the Queries record for the operation: get-/crm/v3/owners/{ownerId}_getById
+public type GetCrmV3OwnersOwnerIdGetByIdQueries record {
+    # Whether to return only results that have been archived.
+    boolean archived = false;
+    "id"|"userId" idProperty = "id";
+};
+
+public type ForwardPaging record {
```

Comment:
Let's add some description for these undocumented records. You can add a description field into the `ForwardPaging` in the final openapi spec (which is flattened and aligned) and regenerate the client and types again. 

Make sure to update that in the sanitations doc.

Check that in other undocumented records as well